src/core/io/src/4C_io_input_field.hpp

```diff
@@ -168,12 +168,18 @@ namespace Core::IO
     /**
      * Access the value of the field for the given @p element index. If the @p element_id
      * is not a valid index, this function will throw an error.
+     * @param element_id The index of the element for which to get the value.
+     * @param field_name Optional name of the field for error messages. This is particularly useful
+     * if multiple input fields are used.
      */
-    [[nodiscard]] const T& at(IndexType element_id) const { return get(element_id, true); }
+    [[nodiscard]] const T& at(IndexType element_id, std::string_view field_name = "") const
```

Comment:
Sorry for only realizing this now: 
Not printing a name inside the '' '' in the error message might be confusing; thus, the default value should likely be something like `"unknown field"`, or you make the parameter non-defaulted.